### PR TITLE
Styiling of the entire page

### PR DESCRIPTION
**Pull request by ghost**
-Margin and padding added to center every element.
-Font size and family changed.
-Buttons styled.
-Code mistakes cleanup.

### CODE REVIEW

**Comment by SevlaMare** [comment]
We are using the Linter locally, Stickler-CI is not recognizing this repository.

File: index.html [hunk 1]
```diff
@@ -57,18 +58,20 @@ <h3>Create an Intuit account</h3>
...
-  <button id="privacity">Privacity Settings</button>
+  <button id="privacity-settings">Privacity Settings</button>
 
 </body>
 
-</html>
\ No newline at end of file
+</html>
+
+<!-- Test comment -->
```
Comment (pelzolga123) [inline index.html]: It's better to delete unnecessary comments.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!

Here are few changes you need to make, before this project can be approved:
* [x] Images on top are not displaying correctly ( just change `/images//images/mint.pngmint.png` to `./images/mint.png` and it will work great)  Right now it looks like this, when you open project without live server:
image
* [x]   Phone `<input>` should be of type tel  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team       
#### Great Job so far, but there is one more thing required. 
### General
- [x] Please fix Stickler errors
### Make changes and submit another code review request.
*Happy coding*

**Pull request closed without merging**